components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -17,6 +18,7 @@ use crate::store::worker::split_config::DEFAULT_SAMPLE_NUM;
 use crate::store::worker::{FlowStatistics, SplitConfig, SplitConfigManager};
 
 pub const TOP_N: usize = 10;
+pub const MAX_RETRY_TIME: i32 = 10000;
```

Comment:
it's too large?

Reply:
Usually , it is possible to sample up to 20 in the qps range, and this may indeed be somewhat large, how about 200？